package.json

```diff
@@ -23,8 +23,12 @@
     "typescript": "3.2.4"
   },
   "devDependencies": {
+    "@babel/core": "7.2.2",
+    "@storybook/react": "4.1.11",
```

Comment:
All `devDependencies` versions should get a caret. I filed https://github.com/mozilla/addons-code-manager/issues/78 to add a test for this.